### DISCUSSION TITLE
Improve turbulence statistics module

### DISCUSSION THREAD

**Discussion by joneuhauser** [Nek5000/nekRS#474] [Ideas]
I'm currently in the process of improving the turbulence statistics module, in order to make most of the capabilities of https://kth-nek5000.github.io/KTH_Framework/dir_89ea5ddbf19ee2b83b7fc9ea72ce96d2.html available in NekRS.

This will include allowing new sections in the par file for activated plugins (in this case, which higher-order statistics and budget terms to write, sampling & writing frequency, whether to write 2D statistics, etc).

According to contributing.md, larger efforts should be coordinated a-priori, so is this something that would be considered for upstream?

**Comment by stgeke** [comment]
Please contact me by email.

**Reply by joneuhauser** [reply]
Hi, I've sent you an email to ker... at anl dot gov on Dec 23. Just wanted to make sure the email address is still up-to-date.

**Comment by stgeke** [comment]
Please use my gmail address
